src/rpc/masternode.cpp

```diff
@@ -43,15 +43,16 @@ static RPCHelpMan masternode_connect()
 {
     std::string strAddress = request.params[0].get_str();
 
-    CService addr;
-    if (!Lookup(strAddress, addr, 0, false))
+    std::optional<CService> addr;
+    if (addr = Lookup(strAddress, 0, false); !addr.has_value()) {
```

Comment:
26261: nit: `addr` is used below so doing it like that
```suggestion
    std::optional<CService> addr{Lookup(strAddress, 0, false)};
    if (!addr.has_value()) {
```
would be less confusing imo

Reply:
Resolved in latest push!